src/api/requestsApi.tsx

```diff
@@ -56,7 +56,7 @@ export interface RPARequest {
   fullPartTime?: "Full" | "Part";
   salaryLow?: number;
   salaryHigh?: number;
-  telework?: "Yes" | "No";
+  telework?: "None" | "Regular-recurring" | "Situational";
   remote?: "Yes" | "No";
```

Comment:
Same comment goes for Remote Work as telework -- needs relocated inside the validateRequest function